pkg/capacity/capacity_test.go

```diff
@@ -1608,9 +1612,13 @@ func str2quantity(str string) *resource.Quantity {
 	return &quantity
 }
 
+var counterMutex sync.Mutex
 var capacityCounter int
 
 func makeCapacity(in testCapacity) *storagev1.CSIStorageCapacity {
+	counterMutex.Lock()
+	defer counterMutex.Unlock()
+	// This is a global counter, that goroutines access causing data race
```

Comment:
How about something like https://gist.github.com/gnufied/f87cb7c574fc74b7b16797fd2d95ac71 ?

Reply:
updated, thanks for help